src/main/java/teammates/ui/pagedata/AdminActivityLogPageData.java

```diff
@@ -402,6 +382,20 @@ private String getActionNameStringFromField(Field field) {
         return actionString;
     }
 
+    public String getQueryKeywordsForInfo() {
+        char delimiter = ',';
+        StringBuffer keywords = new StringBuffer();
+        if (q != null && q.isInfoInQuery) {
```

Comment:
If you use a `if (q == null ||  !q.isInfoInQuery) { return "";}` you can avoid indenting the 'happy path'?